Fix:

error: implicit declaration of function 'safe_snprintf' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'safe_strlcat' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'safe_strlcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- safe_string.h.orig	2003-02-26 22:23:45.000000000 -0600
+++ safe_string.h	2025-08-21 05:51:17.000000000 -0500
@@ -30,12 +30,12 @@
 
 size_t safe_strnlen( const char *str, size_t nchars );
 
-size_t osi_strlcpy( char *dst, const char *src, size_t nchars );
-size_t osi_strlcat( char *dst, const char *src, size_t nchars );
+size_t safe_strlcpy( char *dst, const char *src, size_t nchars );
+size_t safe_strlcat( char *dst, const char *src, size_t nchars );
 
-size_t osi_vsnprintf( char *str, size_t nchars, const char  *format,
+size_t safe_vsnprintf( char *str, size_t nchars, const char  *format,
                       va_list ap );
 
-size_t osi_snprintf( char *str, size_t nchars, const char *format, ... );
+size_t safe_snprintf( char *str, size_t nchars, const char *format, ... );
 
 #endif
